{"name": "测试用例描述：查询客户数据", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'iter'", "trace": "self = <testcase.test_one.Testmysql object at 0x0000022A2120DEF0>, action = None\n\n    @allure.story('用户故事描述：查询数据')  # 模块的功能\n    @allure.description('测试描述：查询数据是否正确')\n    @allure.title('测试用例描述：查询客户数据')\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_03(self,action):\n        db = pymysql.connect(host = '192.168.0.219',port =3306,db='test',user='root',passwd='123',charset='utf8')\n        info = db.cursor()\n        info.execute('select id,username from sec_custinfo')\n        data = info.fetchall()\n        ret =[]\n>       for one,two in data.iter():\nE       AttributeError: 'tuple' object has no attribute 'iter'\n\ntest_one.py:21: AttributeError"}, "description": "测试描述：查询数据是否正确", "start": 1651661154121, "stop": 1651661154124, "uuid": "0d8bd030-10c6-4ae3-9d08-e10abf669cb3", "historyId": "e80dcb360667126e504de2bd04d2fb61", "testCaseId": "673d513ad150bc91b8f870520ad0cdeb", "fullName": "test_one.Testmysql#test_03", "labels": [{"name": "feature", "value": "数据库"}, {"name": "severity", "value": "blocker"}, {"name": "epic", "value": "*************测试描述*************"}, {"name": "story", "value": "用户故事描述：查询数据"}, {"name": "suite", "value": "test_one"}, {"name": "subSuite", "value": "Testmysql"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "4980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_one"}]}